### PR TITLE
[NT-557] Hide "Other payment methods" label when Apple Pay button is hidden

### PR DESCRIPTION
**Pull request by ifbarrera**
# 📲 What

Hides the "Other payment methods" label above stored cards when the Apple Pay button is hidden.

# 🤔 Why

It's a bit strange to say "Other" payment methods when there is only one type of payment method available to the user.

# 🛠 How

Hiding the label when the Apple Pay button is hidden.

# 👀 See

Trello, screenshots, external resources?

| After 🦋| Before 🐛 |
| --- | --- |
| IMG_D5960EBB01E7-1 | Simulator Screen Shot - iPhone 8 - 2019-11-11 at 18 09 53 |

# ♿️ Accessibility 

N/A

# 🏎 Performance

N/A

# ✅ Acceptance criteria

- [ ] Run the app on a device that doesn't have apple pay set up, or alternatively find a project in a country that doesn't support Apple Pay (ex. Mexico). Navigate to the pledge screen - you should not see the label that says "Other payment methods". The Apple Pay button should also be hidden.

### CODE REVIEW

**Comment by justinswart** [comment]
Holding on this based on our Slack convo.

File: Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift [hunk 1]
```diff
@@ -95,10 +96,14 @@ final class PledgePaymentMethodsViewController: UIViewController {
       |> checkoutBackgroundStyle
 
     _ = self.rootStackView
-      |> rootStackViewStyle
+      |> verticalStackViewStyle
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift]: Not sure what the point of this style is, I see we've had it for a while but it just sets the axis to vertical 🤷‍♂

File: Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift [hunk 2]
```diff
@@ -248,12 +253,12 @@ private let titleLabelStyle: LabelStyle = { label in
...
-private let topSectionStackViewStlye: StackViewStyle = { stackView in
+private let applePaySectionStackViewStyle: StackViewStyle = { stackView in
   stackView
-    |> \.axis .~ NSLayoutConstraint.Axis.vertical
-    |> \.spacing .~ Styles.grid(3)
+    |> verticalStackViewStyle
+    |> checkoutSubStackViewStyle
     |> \.isLayoutMarginsRelativeArrangement .~ true
-    |> \.layoutMargins .~ UIEdgeInsets(leftRight: CheckoutConstants.PledgeView.Inset.leftRight)
+    |> \.layoutMargins .~ .init(leftRight: Styles.grid(4))
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift]: What this `4` before?

File: Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift (hunk 1 above)
Comment (ifbarrera) [reply Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift]: Ask @dusi 😆

**Pull request merged**